base/controller.go

```diff
@@ -461,10 +464,25 @@ func (rd *RegionDescription) TagsFound(tl interface{}) bool {
 			}
 		}
 		if numberOfNegativeMatches == 0 {
-			return true
+			return tags, true
 		}
 	}
-	return false
+	return tags, false
+}
+
+func TagsToString(tags []*TagDescription) *string {
+	result := "N/A"
+	if len(tags) < 1 {
+		return &result
+	}
+
+	tl := []string{}
+	for _, tag := range tags {
```

Comment:
Is the order of tags from the AWS API guaranteed? otherwise we might want to sort the tags to ensure they are stable.

Reply:
According to output yes. But i could not find an explicit statement in the docs about this,